kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/modem/SupportedUsbModems.java

```diff
@@ -87,63 +92,54 @@ public static void installModemDrivers(CommandExecutorService executorService) {
         }
     }
 
-    public static SupportedUsbModemInfo getModem(String vendorId, String productId) {
-        if (vendorId == null || productId == null) {
+    public static SupportedUsbModemInfo getModem(UsbDevice usbDevice) {
```

Comment:
I believe those changes here are a sort of breaking change. Is the manifest updated to export a new major? Or, if we can still be retro-compatible, can we have method overloading with deprecation?

Reply:
I can move org.eclipse.kura.linux.net.modem from 1.1.1 to 2.0.0

Reply:
Is it possible to deprecate the old and add the new method?